src/shared/SolidLabel/SolidLabel.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Label, LabelProps } from "@patternfly/react-core";
+import { AclResourceType } from "../../utils";
+import "./SolidLabel.css";
+
+export type SolidLabelProps = {
+  variant: AclResourceType;
+};
+
+type Variant =
+  | {
+      labelColor: LabelProps["color"];
+      content: string;
+    }
+  | undefined;
```

Comment:
```suggestion
```

Always favor stricter types. We don't need to support unknown groups, so we don't need to allow for unknown variants.